src/test/resources/features/apiaddexperience.feature

```diff
@@ -0,0 +1,14 @@
+@ApiAddExperience
+Feature: add experience
+
+  @smoke @regression @api @unhappy
+  Scenario: add experience unhappy
+    Given [API] user is logged in
+      | username               | password |
+      | lixiasun2000@gmail.com | Lsun1688 |
+    Then user enters incorrect add experience inputs
```

Comment:
- This step should just enter experience details, regardless if it's valid or invalid.
- All API steps should have `[API]` prefix